### ISSUE TITLE
Cli argument parsing seems broken

### ISSUE THREAD

**Issue by serkonda7** [near/create-near-app#2111]
The following command leads to an error. However the order, flags and arguments match the required scheme.
```
npx create-near-app my-app --contract ts --frontend vanilla --tests ts
```

```
Arguments error
Run npx create-near-app without arguments, or use:
npx create-near-app <projectName> --frontend next|vanilla|none --contract rs|ts|none --tests rs|ts|none
```

**Comment by gagdiez** [comment]
Nice catch, addressed by #2113

**Comment by serkonda7** [comment]
Thanks. The error is clear now
